helm/coredns-app/templates/test/test-runner.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   # Bash automated testing system
   # https://github.com/bats-core/bats-core
   - name: test-framework
-    image: {{ .Values.test.image.registry }}/giantswarm/bats:0.4.0
+    image: {{ .Values.image.registry }}/giantswarm/bats:0.4.0
```

Comment:
I don't love these helm tests because there is a lot of bash "magic". But anyway we should only have the `image.registry` value once.

FYI @sslavic As you were looking at` terratest` to have a better testing experience for charts.